docs/en/latest/admin-api.md

```diff
@@ -1497,6 +1497,102 @@ HTTP/1.1 200 OK
 {"key":"\/apisix\/secrets\/vault\/test2","value":{"id":"vault\/test2","token":"apisix","prefix":"apisix","update_time":1669625828,"create_time":1669625828,"uri":"http:\/\/xxx\/get"}}
 ```
 
+When `{secretmanager}` is `aws`:
+
+| Parameter | Required | Type | Description | Example |
+| --- | --- | --- | --- | --- |
+| access_key_id | True | string | AWS Access Key ID |  |
+| secret_access_key | True | string | AWS Secret Access Key |  |
+| session_token | False | string | Temporary access credential information |  |
+| region | False | string | AWS Region |  |
+| endpoint_url | False | URI | AWS Secret Manager URL | https://secretsmanagerus-east-1.amazonaws.com |
```

Comment:
Please make the content consistent: 

<img width="720" alt="image" src="https://github.com/user-attachments/assets/ef71523a-5e4e-42b1-8962-6921c31e52c3">

Reply:
Fixed.